Rx/v2/src/rxcpp/rx-util.hpp

```diff
@@ -416,6 +416,15 @@ struct less
         { return std::forward<LHS>(lhs) < std::forward<RHS>(rhs); }
 };
 
+template <class T>
+struct ret
+{
+    template <class LHS>
+    auto operator()(LHS&& lhs) const
```

Comment:
I think that if you remove the lhs and submit that to the same branch it will pass the build tests.